server/config/topic_page/equity/US_Places.textproto

```diff
@@ -140,10 +140,10 @@ categories {
   }
 
   stat_var_spec {
-    key: "max_temp_prediction"
+    key: "max_temp_projection"
     value {
       stat_var: "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585"
-      name: "MaxTemp Ensemble GaussianMixture 5PctProb Greater Atleast1DayAYear CMIP6 SSP245"
+      name: "Projected max temp (5% probability, decadal, SSP245)"
```

Comment:
super nit: s/max temp/Max Temperature/ to match the topics elsewhere

Reply:
done